geopandas/datasets/__init__.py

```diff
@@ -18,6 +18,12 @@ def get_path(dataset):
         The name of the dataset. See ``geopandas.datasets.available`` for
         all options.
 
+    Examples
+    --------
+    >>> geopandas.datasets.get_path("naturalearth_lowres")  # doctest: +SKIP
+    '/opt/miniconda3/envs/geo_env/lib/python3.8/site-packages/geopandas/datasets/\
```

Comment:
```suggestion
    '.../python3.8/site-packages/geopandas/datasets/\
```

(since it's skipped anyway, can maybe simplify it a bit to not show the system specific part)